Assets/Scripts/CPUConfig.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "CPUConfig", menuName = "Game/CPUConfig")]
+public class CPUConfig : ScriptableObject
+{
+    [System.Serializable]
+    public class ModeSettings
+    {
+        public string modeName;     // モード名 (例: Easy, Normal, Hard)
+        public float moveSpeed;     // 横移動速度
+        public float rotationSpeed; // 回転速度
+    }
+
+    [Header("CPU Modes Settings")]
+    public ModeSettings[] modeSettings;
```

Comment:
細かいけどこっちの方が安全、Unity上に表示しつつ非公開にできる。privateになったので変数名の前に"_"つけよう。
```suggestion
    [SerializeField] private ModeSettings[] _modeSettings;
```